config/config.json

```diff
@@ -20,19 +19,17 @@
         "version": "1.0.0",
         "plugins": [
             {
-                "name":"notaryv2",
-                "artifactTypes" : "application/vnd.cncf.notary.v2.signature",
+                "name": "notaryv2",
+                "artifactTypes": "application/vnd.cncf.notary.v2.signature",
                 "verificationCerts": [
-                    "<cert folder>"
-                  ]
+                    "<certificate file>"
```

Comment:
this file gets added as part of image https://github.com/deislabs/ratify/blob/main/httpserver/Dockerfile#L26

this should give more information if someone doesn't modify this

Reply:
The config is overridden through the config map during helm install. Do you suggest the placeholder to be path ot certificate file?

Reply:
I meant what happens if someone deploys outside Helm. I think `"<certificate file>"` can be text can be changed to clarify that user needs to overwrite to config or don't add config to the image (i am guessing it'll error if config is not found) and have instructions to read from a configmap.

Reply:
Thanks @sozercan  

@sajayantony , I updated the config to include sample verifier . Added another config with notary and sbom verifiers 
https://github.com/mnltejaswini/ratify/commit/6161989882d7d952882f2d8f1115d20dd179c3cf

Reply:
@mnltejaswini - Are you planning to send a PR for those samples. If so I can close this PR.

Reply:
Yes please. I will send another PR with suggestions proposed by Sertac